addons/l10n_es/data/template/account.fiscal.position-es_common.csv

```diff
@@ -33,15 +33,15 @@
 "","","","","","","","","","","","account_common_7050","account_common_7051"
 "fp_extra","4","Extra-community","Régimen Extracomunitario","Règim extracomunitari","1","","","","account_tax_template_p_iva0_s_bc","account_tax_template_p_iva0_ibc","",""
 "","","","","","","","","","account_tax_template_p_iva0_s_sc","account_tax_template_p_iva0_isc","",""
-"","","","","","","","","","account_tax_template_p_iva4_bc","","",""
+"","","","","","","","","","account_tax_template_p_iva4_bc","account_tax_template_p_iva0_ibc_group","",""
```

Comment:
This is wrong.  This should map to the exento tax...

Reply:
That is the Exempt DUA tax, maybe the external_id name is wrong or i am misunderstanding something?

Reply:
Oh wow, you are right.  It is just that the account_tax_template_p_iva0_ibc_group name is confusing.  As it is not even a tax group.

Reply:
I have changed it's name to account_tax_template_p_iva0_dua_exempt, does it seem better?

Reply:
Yes.  It is not really iva0 as well, so I wonder if we should remove the _iva0 part.